packages/at_client/lib/src/service/sync_service_impl.dart

```diff
@@ -984,76 +987,71 @@ class SyncServiceImpl implements SyncService, AtSignChangeListener {
     return unCommittedEntryBatch;
   }
 
-  void _setMetaData(UpdateVerbBuilder builder, serverCommitEntry) {
+  void _setMetadataFromCommitEntry(Metadata md, Map serverCommitEntry) {
```

Comment:
Renamed for readability, and pass a Metadata argument rather than an UpdateVerbBuilder argument so we can use this function in multiple places.